extension/chrome/elements/compose-modules/compose-storage-module.ts

```diff
@@ -114,8 +138,8 @@ export class ComposeStorageModule extends ViewModule<ComposeView> {
    *    discussion: https://github.com/FlowCrypt/flowcrypt-browser/pull/3898#discussion_r686244628
    */
   public lookupPubkeyFromKeyserversAndUpsertDb = async (
-    email: string, name: string | undefined, existingContact: Contact | undefined
-  ): Promise<Contact | "fail"> => {
+    email: string, name: string | undefined, existingPubKey: PubKeyInfo | undefined
+  ): Promise<PubKeyInfo | "fail" | undefined> => {
```

Comment:
I guess this should be `Promise<PubkeyInfo[]> |  'fail'`?

Reply:
UPD: it should be `Promise<'fail' | 'success'> or `Promise<boolean>` as re-reading the storage perplexes some of the callers (`lookupPubkeyFromKeyserversThenOptionallyFetchExpiredByFingerprintAndUpsertDb`)

Reply:
Why? It must return a key. How this can be changed to just boolean?